README.md

```diff
@@ -75,6 +75,72 @@ python manage.py test
 flake8 .
 ```
 
+### Windows 10 Users
+ Please install and set up the following packages first, upgrade if you find the package is already installed:
+
+* Download and install [Python3](https://www.python.org/downloads/) and ensure to 'Add path' while installing. Run the 'python3 -V' command to see the version you have installed.
+
+* Download and install [pip](https://pip.pypa.io/en/latest/installing/) and follow the installing with get-pip.py instructions.
+
+* [PostgreSQL](https://www.postgresql.org/download/windows/) (Ensure the server is running).
```

Comment:
What were the blockers you faced whilst installing this? How did you bypass them? It's helpful to add those as a sort of nota bene. "If you find that you're experiencing difficulties with installing postgres, do this and that", that sort of thing.

Reply:
After downloading, you may follow these 10 steps as depicted on  https://www.postgresqltutorial.com/install-postgresql/

Reply:
NK can i add a youtube link for ease of installation on the Readme file?

Reply:
hmm, i guess.

Reply:
Since Delphine didnt get the error message "Unable to write to TEMP environment" while installing would it be okay if i still go ahead and input it there as one of the blockers and how i got off it?

Reply:
What was the reason for the error though?